rest-api-spec/src/main/resources/rest-api-spec/test/nodes.info/40_aggs.yml

```diff
@@ -0,0 +1,20 @@
+---
+"node_info test aggregations":
+  - skip:
+      version: " - 7.99.99"
+      reason: "aggregation info only supported in 8.0.0+"
+      features: [arbitrary_key]
+
+
+  - do:
+      nodes.info: {}
+  - set:
+      nodes._arbitrary_key_: node_id
+
+  - do:
+      nodes.info:
+        metric: [ aggregations ]
+
+  # if this test failed because a new aggregation was added, please open an issues in the elastic/telemetry repository
+  # so they can update the mapping accordingly
```

Comment:
this comment seems abit out of place, and probably shouldn't reference a private repo. 

also, I guess I don't understand the output.  is it aggregrations.??.types = ?? . Is there anything that prevents checking for known types

Reply:
I was sure I removed it. This comment is out of place. Good catch.

Reply:
The output looks like this:
```
      "aggregations" : {
        "adjacency_matrix" : {
          "types" : [
            "other"
          ]
        },
        "auto_date_histogram" : {
          "types" : [
            "boolean",
            "date",
            "numeric"
          ]
        },
        "avg" : {
          "types" : [
            "boolean",
            "date",
            "histogram",
            "numeric"
          ]
        },
        "boxplot" : {
          "types" : [
            "histogram",
            "numeric"
          ]
        },
        "cardinality" : {
          "types" : [
            "boolean",
            "bytes",
            "date",
            "geopoint",
            "geoshape",
            "ip",
            "numeric",
            "range"
          ]
        },
        ....
```

and both types and aggs change depending on license that this is ran under.